imod/tests/test_prepare/test_topsystem.py

```diff
@@ -136,7 +136,7 @@ def test_distribute_drn_conductance(
     conductance = zeros_like(elevation) + 1.0
 
     actual_da = distribute_drn_conductance(
-        option, allocated, conductance, top, bottom, k
+        option, allocated, conductance, top, bottom, k, elevation
     )
     actual = take_first_cell_in_xy_plane(actual_da)
```

Comment:
you only compare "actual" and "expected" in the first cell.  So it would make the tests more strict if arrays like "bottom" and "elevation" have a different value in the first cell than in the rest of the array. Otherwise the tests would still pass even if something went wrong in the indexing.

Reply:
By doing this, we would mainly test if the testing utility function ``take_first_cell_in_xy_plane`` does what it's supposed to do. I think the it adds a quite some complexity to the test cases for relatively little gains. I think it is better to allocate our constrained resources to fixing other issues.